src/sentry/static/sentry/app/views/settings/projectAlerts/alertTypeChooser.tsx

```diff
@@ -28,57 +26,58 @@ const IssuesTooltip = ({children}: {children?: React.ReactNode}) => (
 
 const AlertTypeChooser = ({selected, onChange}: Props) => (
   <Container>
-    <TypeCard interactive onClick={() => onChange('metric')}>
+    <TypeCard interactive onClick={() => onChange('issue')}>
       <RadioLabel>
         <Radio
-          aria-label="metric"
-          checked={selected === 'metric'}
-          onChange={() => onChange('metric')}
+          aria-label="issue"
+          checked={selected === 'issue'}
+          onChange={() => onChange('issue')}
         />
-        {t('Metric Alert')}
-        <FeatureBadge type="beta" />
+        {t('Issue Alert')}
       </RadioLabel>
       <p>
         {tct(
-          `Compute aggregates and set thresholds on all errors in your project,
-           regardless of the [note:Sentry Issue].`,
+          `Get notified when [note:individual Sentry Issues] match your alerting criteria.`,
           {note: <IssuesTooltip />}
         )}
       </p>
       {!selected && (
         <BulletList>
           <li>
-            {t('Overall error volume')}
-            <Example>{t('A broken service is affecting more than X users')}</Example>
+            {t('New or regressed issues')}
+            <Example>{t('New issue on the checkout page')}</Example>
           </li>
           <li>
-            {t('Events across issues')}
-            <Example>{t('100 or more errors with "database" in the title')}</Example>
+            {t('Issue frequency')}
+            <Example>{t('Issue affecting more than X users')}</Example>
           </li>
         </BulletList>
       )}
     </TypeCard>
-    <TypeCard interactive onClick={() => onChange('issue')}>
+    <TypeCard interactive onClick={() => onChange('metric')}>
       <RadioLabel>
         <Radio
-          aria-label="issue"
-          checked={selected === 'issue'}
-          onChange={() => onChange('issue')}
+          aria-label="metric"
```

Comment:
This might break tests

Reply:
It didn't locally, but I'll see it on the ci reports if it does